it/v1/endpoints/CreateAndAmendEmploymentExpensesControllerISpec.scala

```diff
@@ -89,14 +89,6 @@ class CreateAndAmendEmploymentExpensesControllerISpec extends IntegrationBaseSpe
           response.json shouldBe Json.toJson(RuleTaxYearNotSupportedError)
         }
 
-        s"a taxYear that hasn't ended is provided" in new NonTysTest {
-          override val taxYear: String = getCurrentTaxYear
-
-          val response: WSResponse = await(request().put(requestBodyJson))
-          response.status shouldBe BAD_REQUEST
-          response.json shouldBe Json.toJson(RuleTaxYearNotEndedError)
-        }
-
         s"an invalid amount is provided" in new NonTysTest {
```

Comment:
Could remove the "s" in front as we are not using variables. For all relevant tests on this file